provider/core/src/serde/mod.rs

```diff
@@ -24,16 +24,16 @@
 
 pub mod borrow_de_utils;
 mod de;
-#[cfg(feature = "serialize")]
+#[cfg(feature = "serde")]
```

Comment:
I feel like these can all be `datagen`?

Reply:
Hmm, I'd rather not make larger changes here. I suspect it can be, we probably want to move more into `datagen`, but we should handle that separately.